.docsettings.yml

```diff
@@ -16,6 +16,18 @@ required_readme_sections:
   - ^Troubleshooting$
   - ^Next steps$
   - ^Contributing$
+
+known_presence_issues:
+  - ['HISTORY.rst','#4554']
+  - ['sdk/cognitiveservices/azure-cognitiveservices-vision-nspkg/HISTORY.rst','#4554']
+  - ['sdk/cognitiveservices/azure-cognitiveservices-language-nspkg/HISTORY.rst','#4554']
+  - ['sdk/keyvault/azure-keyvault-nspkg/HISTORY.rst','#4554']
+  - ['sdk/template/azure-template/HISTORY.rst','#4554']
+  - ['sdk/cognitiveservices/azure-cognitiveservices-knowledge-nspkg/HISTORY.rst','#4554']
+  - ['sdk/cognitiveservices/azure-cognitiveservices-nspkg/HISTORY.rst','#4554']
+  - ['sdk/cognitiveservices/azure-cognitiveservices-search-nspkg/HISTORY.rst','#4554']
+  - ['sdk/core/azure-nspkg/HISTORY.rst','#4554']
```

Comment:
@Scebd are any of these track 2 packages, in which case I should start a History file for them.